app/controllers/sessions_controller.rb

```diff
@@ -15,6 +15,6 @@ def create
   def destroy
     session.clear
     flash[:success] = 'You have successfully logged out!'
-    redirect_to '/'
+    redirect_to welcome_index_path
```

Comment:
minor, but I'd prefer to call this  'welcome_path' - I don't think there's a truly restful standard for the root directory, but 'index' seems unnecessary.

Reply:
has to have the index... it's populated with rails_helpers